section4/exercises/classMethods.js

```diff
@@ -28,13 +28,27 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, top) {
     this.protein = protein;
     this.base = base;
+    this.toppings = top;
   }
-
-  // ADD CODE
-};
-
-// ADD CODE
-
+changeProtein(cp){
+  this.protein = cp
+}
+addTopping(tp) {
+  this.toppings = this.toppings + tp
```

Comment:
Isn't `this.toppings` an array?

Reply:
Yes it is an array. The addToppings method is created in order for the array to add the parameter. I found this method better than hard coding it, because this way. I can change the perimeters that appear along the toppings at all times. If there is an easier way or cleaner way of doing it, I would appreciate if you could tell me.